content/tracing/visualization/services_map.md

```diff
@@ -31,33 +22,33 @@ The Service Map decomposes your application into all its component services and
 
 ## Setup
 
-The Service Map visualizes data that is being collected by Datadog APM. No setup is required.
+The Service Map visualizes data collected by Datadog APM. Setup is not required to view services. To see connected services, enable [distributed tracing][3].
 
 ## Ways to use it
 
-We built the Service Map not only to give you an overview of all your services and their health, but also to cut through noise and isolate problem areas. You can also access other telemetry collected by Datadog directly from this view.
+The Service Map was built to provide an overview of your services and their health and to cut through noise and isolate problem areas. Also, you can access other telemetry collected by Datadog directly from this view.
```

Comment:
I think we should break the first sentence here in 2 instead of having 3 ands. Everything else looks good 👍